packages/quantic/cypress/e2e/default-2/smart-snippet/smart-snippet-expectations.ts

```diff
@@ -5,13 +5,22 @@ import {
   SmartSnippetSelectors,
 } from './smart-snippet-selectors';
 
+function logCustomSmartSnippetEvent(event: string) {
+  cy.wait(event)
+    .then((interception) => {
+      const analyticsBody = interception.request.body;
+      expect(analyticsBody).to.have.property('eventType', 'smartSnippet');
+    })
+    .logDetail(`should log the '${event}' UA event`);
+}
+
 function smartSnippetExpectations(selector: SmartSnippetSelector) {
   return {
     displaySmartSnippetCard: (display: boolean) => {
       selector
         .smartSnippetCard()
         .should(display ? 'exist' : 'not.exist')
-        .log('should display the smart snippet');
+        .log(`${should} display the smart snippet`);
```

Comment:
`should` is a function, and `logDetail` should be used instead of `log`. The other expectations should be fixed the same way.

```suggestion
        .logDetail(`${should(display)} display the smart snippet`);
```

Reply:
Thanks I'll fix them all!